libs/shared/ui/nuverial/src/lib/components/button/button.component.html

```diff
@@ -5,8 +5,8 @@
     [attr.aria-describedby]="ariaDescribedBy"
     [attr.aria-label]="ariaLabel"
     [color]="$any(colorTheme)"
-    [disabled]="disabled"
-    [ngClass]="{uppercaseText: uppercaseText}"
+    [disabled]="loading? true: disabled"
```

Comment:
This is a slightly better way to say the same thing

```suggestion
    [disabled]="loading || disabled"
```

Reply:
Will change it in the next PR